recipes-core/customizations-example/files/board-configuration.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description="IOT2050 Board Configuration"
+Description=IOT2050 Board Configuration
```

Comment:
Style fix or more? This is missing in your commit message.

Reply:
Moved to https://github.com/siemens/meta-iot2050/pull/397